pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.black]
 line-length = 120
-target-version = ['py39', 'py310', 'py311', 'py312', 'py313']
```

Comment:
py313 did not work for me - thoughts @pedrocamargo

Reply:
It is a Black version thing, I believe. It works well for me.

https://black.readthedocs.io/en/stable/change_log.html